src/sim_config.cpp

```diff
@@ -216,20 +216,30 @@ namespace Nos3
                 _config.put("nos3-configuration.common.log-config-file", "sim_log_config.xml");
             }
 
```

Comment:
If no arguments passed in no error reported.

Reply:
Concur.  Will make error/help more explicit.